Namespace.cpp

```diff
@@ -27,16 +27,16 @@ CNamespace::~CNamespace()
 {
 }
 
-void CNamespace::Serialize(std::ifstream& ifstream)
+void CNamespace::Serialize(CFalloutScriptData& scriptData)

```

Comment:
You should only pass the thing you're using to the function. Otherwise such interface implies function can read and write, which isn't the case.